shared/i18n/locales/en_US/translation.json

```diff
@@ -1,535 +1,548 @@
-  "Disconnect": "Disconnect",
-  "Connect Outline collections to Slack channels and messages will be automatically posted to Slack when documents are published or updated.": "Connect Outline collections to Slack channels and messages will be automatically posted to Slack when documents are published or updated.",
-  "Connected to the <em>{{ channelName }}</em> channel": "Connected to the <em>{{ channelName }}</em> channel",
-  "Connect": "Connect",
-  "New token": "New token",
-  "You can create an unlimited amount of personal tokens to authenticate\n          with the API. Tokens have the same permissions as your user account.\n          For more details see the <em>developer documentation</em>.": "You can create an unlimited amount of personal tokens to authenticate\n          with the API. Tokens have the same permissions as your user account.\n          For more details see the <em>developer documentation</em>.",
-  "Tokens": "Tokens",
-  "Create a token": "Create a token",
-  "Zapier": "Zapier",
-  "Zapier is a platform that allows Outline to easily integrate with thousands of other business tools. Head over to Zapier to setup a \"Zap\" and start programmatically interacting with Outline.'": "Zapier is a platform that allows Outline to easily integrate with thousands of other business tools. Head over to Zapier to setup a \"Zap\" and start programmatically interacting with Outline.'",
-  "Open Zapier": "Open Zapier",
-  "You’ve not starred any documents yet.": "You’ve not starred any documents yet.",
-  "There are no templates just yet.": "There are no templates just yet.",
-  "You can create templates to help your team create consistent and accurate documentation.": "You can create templates to help your team create consistent and accurate documentation.",
-  "Trash is empty at the moment.": "Trash is empty at the moment.",
-  "Are you sure? Deleting your account will destroy identifying data associated with your user and cannot be undone. You will be immediately logged out of Outline and all your API tokens will be revoked.": "Are you sure? Deleting your account will destroy identifying data associated with your user and cannot be undone. You will be immediately logged out of Outline and all your API tokens will be revoked.",
-  "<em>Note:</em> Signing back in will cause a new account to be automatically reprovisioned.": "<em>Note:</em> Signing back in will cause a new account to be automatically reprovisioned.",
-  "Delete My Account": "Delete My Account",
-  "You joined": "You joined",
-  "Joined": "Joined",
-  "{{ time }} ago.": "{{ time }} ago.",
-  "Edit Profile": "Edit Profile",
-  "{{ userName }} hasn’t updated any documents yet.": "{{ userName }} hasn’t updated any documents yet."
-}
+{

```

Comment:
Not sure why GitHub thinks this entire file is changed? 🤔

Reply:
Yeah it got me a little confused. Tried some things but it still thinks everything changed lol

Reply:
Are you on Windows? Looks like line-endings changed:

![image](https://user-images.githubusercontent.com/380914/126909517-dd077fc0-d509-44a9-b446-c1b25f6941b3.png)

Reply:
Windows line endings always giving me trouble 😑. I think it worked this time.